service/src/main/frontend/.env

```diff
@@ -0,0 +1 @@
+ENDPOINT="https://localhost:8643/dictionary/data/v2/"
```

Comment:
I am not sure I understand why we need this.

Reply:
I kept the endpoint in the `.env` since I thought we shouldn't hard code it, I can save it as a variable in my typescript file since it doesn't contain anything else.

Reply:
This is removed and is added in dynamically  now.